hikari/applications.py

```diff
@@ -335,6 +337,18 @@ def avatar_url(self) -> typing.Optional[files.URL]:
     def default_avatar_url(self) -> files.URL:
         return self.user.default_avatar_url
 
+    @property
+    def banner_hash(self) -> undefined.UndefinedOr[str]:
+        return self.user.banner_hash
+
+    @property
+    def banner_url(self) -> typing.Optional[files.URL]:
+        return self.user.banner_url
+
+    @property
+    def accent_color(self) -> undefined.UndefinedOr[colors.Color]:
+        return self.user.accent_color
+
```

Comment:
UK is mad at you now :(

Reply:
nah it isn't, the colour alias is implemented on PartialUser

Reply:
Sorry UK😅